zstacklib/zstacklib/utils/objgraph.py

```
@@ -0,0 +1,960 @@
+    name = objtype.__name__
+    module = getattr(objtype, '__module__', None)
+    if module:
+        return '%s.%s' % (module, name)
+    else:
+        return name
+
+
+def _safe_repr(obj):
+    try:
+        return _short_repr(obj)
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)